### ISSUE TITLE
OSX 10.7 Crash

### ISSUE THREAD

**Issue by bceverly** [Fingertips/passengerpane#67]
The prefpane crashes under the GM of Lion for what it's worth...

**Comment by alloy** [comment]
Did you build from the current master branch? Or are you using a release version? (e.g. v1.3) Because the former is known to build on older Lion previews, the latter is not supported on Lion.

**Comment by agiletortoise** [comment]
Can you provide a build of the current Master, or instructions to do the build properly?  I'm not a Mac dev, I've got the project up in Xcode 4.1, but am not sure how to get a build of the prefPane that I can actually install.

**Comment by eric** [comment]
I'd be interesting in how to build it as well.

**Comment by joris** [comment]
Could you just update the stable version that's on your website?

**Comment by Manfred** [comment]
Yes, but the rewrite of the Passenger Pane is kinda alpha right now. We will try to release a beta version soon.

**Comment by joris** [comment]
okido!

**Comment by agiletortoise** [comment]
I cloned master, opened the project in Xcode 4.1/Lion, let Xcode update the targets (automatically), did build and am using the "ppain" command line tool without issue now, FYI.  I did like having the prefPane though.

**Comment by eric** [comment]
I found running `xcodebuild` on the command line built the prefpane for me.

**Comment by cthree** [comment]
clone it, cd to it, run xcodebuild in it, open it/build/Release folder with finder, double click Passenger.prefPane and you're set. Worked for me without a hitch.

**Comment by Bhuwan** [comment]
Same issue here.  Building locally not working either.

**Comment by Preacher** [comment]
Thanks @cthree.

**Comment by jeffpoulton** [comment]
I'm trying to build against master on a clean Lion install. I'm getting this error, both from within xcode and on the command line as eric suggested. Any ideas?

error: can't exec '/Developer/Library/Xcode/PrivatePlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer/Library/Xcode/Plug-ins/LLVM GCC 4.2.xcplugin/Contents/Resources/gcc-4.2' (No such file or directory)

**Comment by Phiction** [comment]
@cthree /cheers

**Comment by kellym** [comment]
@jeffpoulton i had the same issue. for some reason the compiler didn't get a symlink in that folder... i just ran this in my terminal and retried the build, it worked fine:

ln -s /Developer/usr/bin/gcc-4.2 /Developer/Library/Xcode/PrivatePlugIns/Xcode3Core.ideplugin/Contents/SharedSupport/Developer/Library/Xcode/Plug-ins/LLVM\ GCC\ 4.2.xcplugin/Contents/Resources/

**Comment by jeffpoulton** [comment]
@kellym, thx...worked as prescribed!

**Comment by twestonkendall** [comment]
thanks @cthree

**Comment by drnic** [comment]
Will there be a 1.4 release that works for Lion?

**Comment by Manfred** [comment]
Yes, but unfortunately Eloy and I are very busy and we haven't found the time to fix some visual problems related to the view with the hostnames.
